src/data/mv_polynomial.lean

```diff
@@ -831,6 +843,17 @@ finset.sup_le $ assume n hn,
     { assume a b₁ b₂, refl }
   end
 
+lemma total_degree_pow (a : mv_polynomial σ α) (n : ℕ) :
+  (a^n).total_degree ≤ n * a.total_degree :=
```

Comment:
```suggestion
  (a ^ n).total_degree ≤ n * a.total_degree :=
```
I don't really care too much but according to [style.md](https://github.com/leanprover-community/mathlib/blob/master/docs/contribute/style.md#structuring-definitions-and-theorems) there should be spaces around all `^` and `+`. The whole file seems to be very inconsistent about it.